File: app/src/test/java/science/apolline/TestUtility.java
package science.apolline;

/**
 * Created by sparow on 10/9/17.
 */

import java.util.Map;

public class TestUtility {


    public static String getInfluxDBIP() {
        String ip = "127.0.0.1";

        Map<String, String> getenv = System.getenv();
        if (getenv.containsKey("INFLUXDB_IP")) {
            ip = getenv.get("INFLUXDB_IP");
        }

        return ip;
    }



    public static String getInfluxDBPORT(boolean apiPort) {
        String port = "8086";

        Map<String, String> getenv = System.getenv();
        if(apiPort) {
            if (getenv.containsKey("INFLUXDB_PORT_API"))
                port = getenv.get("INFLUXDB_PORT_API");
        }
        return port;
    }
}

File: app/src/test/java/science/apolline/TestUtility.kt
package science.apolline

/**
 * Created by sparow on 10/9/17.
 */
object TestUtility {

    val influxDBIP: String
        get() {
            var ip = "localhost"

            val getenv = System.getenv()
            if (getenv.containsKey("INFLUXDB_IP")) {
                ip = getenv["INFLUXDB_IP"].toString()
            }

            return ip
        }


    fun getInfluxDBPORT(apiPort: Boolean): String {
        var port = "8086"

        val getenv = System.getenv()
        if (apiPort) {
            if (getenv.containsKey("INFLUXDB_PORT_API"))
                port = getenv["INFLUXDB_PORT_API"].toString()
        }
        return port
    }
}

